docs/configuration.md

```diff
@@ -135,7 +135,7 @@ telemetries: [
 | urlsToInclude | RegExp[] | `[/.*/]` | A list of HTTP request (`XMLHttpRequest` or `fetch`) URLs. These requests will be recorded, unless explicitly excluded by `urlsToExclude`. |
 | urlsToExclude | RegExp[] | `[]` | A list of HTTP request (`XMLHttpRequest` or `fetch`) URLs. These requests will not be recorded. |
 | stackTraceLength | Number | `1000 ` | The number of characters to record from a JavaScript error's stack trace (if available). |
-| recordAllRequests | boolean | `false` | By default, only HTTP failed requests (i.e., those with network errors or status codes which are not 2xx) are recorded. When this field is `true`, the http telemetry will record all requests, including those with successful 2xx status codes. <br/><br/>This field does **does not apply** to X-Ray traces, where all requests are recorded. |
+| recordAllRequests | boolean | `true` | By default, all HTTP requests are recorded. When this field is `false`, http telemetry will only record failed requests (i.e., those with network errors or status codes which are not 2xx). <br/><br/>This field does **does not apply** to X-Ray traces, where all requests are recorded. |
```

Comment:
issue: PRs should be [*"one self-contained change"*](https://google.github.io/eng-practices/review/developer/small-cls.html).

I believe that the best practice when using [GitHubFlow](https://gitversion.net/docs/reference/modes/mainline) is to merge #412 into a branch until we are preparing to release the next major version.

I've created a release branch for the next major revision -- `release/2.0.x`. We can merge #412 into that branch and rebase this change on top of it. You'll need to update the target branch for #412 before it can be merged in.

Reply:
Target branch for #412 has been updated to release/2.0.x